### ISSUE TITLE
Stopped working yesterday 

### ISSUE THREAD

**Issue by hartmood** [ldotlopez/ha-ideenergy#46]
My installation worked for many months without issues. It stopped reading the latest data on the 31st of May. I still can log in to my account with the mobile app and current data is shown there. 

Any ideas?

**Comment by luca-angemi** [comment]
hi,  

the website is broken since yesterday. no data coming in.

![image](https://github.com/ldotlopez/ha-ideenergy/assets/9977955/aa1c021a-75fa-4d23-aa18-7b3569334d8a)

**Comment by sirdir1972** [comment]
The website sometimes gives this message but it still works. 
What’s stranger is this:
The device is disabled by Config entry. Disabled devices will not be shown and entities belonging to the device will be disabled and not added to Home Assistant.
(i-de plugin)
Of course that’s not true.

**Comment by luca-angemi** [comment]
That is if you add the integration now, right?

Since I've had the intergration before the issue started my device is not disabled and the enties are active.

**Comment by sirdir1972** [comment]
Hm, not knowingly, the integration was added long ago.

**Comment by luca-angemi** [comment]
and suddenly your device become unavailable? that's strange!

**Comment by sirdir1972** [comment]
Well, that‘s how it seems to me… What I did is trying to re-add the integration but that just said it already is installed. And as I said, the integration claims it‘s deactivated by config. Sent from my iPadOn 2 Jun 2023, at 15:35, Luca Angemi ***@***.***> wrote:﻿
and suddenly your device become unavailable? that's strange!

**Comment by luca-angemi** [comment]
Yep, that's what I thought. 

If I'm not mistaken, re-adding the integration it first unloads and later loads the integration. 
If during the loading process you're not able to connect to the device, the device won't be activated.

**Comment by hartmood** [comment]
I deleted and re-added the integration. Still, no new data is coming in.

**Comment by luca-angemi** [comment]
Same on this side. I mean I didn't delete the integration but no data is coming in since a few days now. The website still shows the same errors when trying to connect manually to the meter.

**Comment by ad-ha** [comment]
Hi, All,

There's  an issue with I-DE directly, not the integration. I-DE has been changing the web and there were some glitches. It's not even possible (at least in my case) to access the meter from their web.

![image](https://github.com/ldotlopez/ha-ideenergy/assets/59612788/79e57fc4-9fdc-439c-8609-4807b2702faa)
This was just now.

The historical data seems to be getting up to speed on the web, but no instant access has been possible since 31 may.

Just wait for i-DE to sort their issues out and the integration shall get data back. Unless they are also changing things within the addresses, which so far does not seem to be the case.

**Comment by hartmood** [comment]
I filed a reclamation. 
![image](https://github.com/ldotlopez/ha-ideenergy/assets/50808701/0e7f9510-55ea-4893-bf5e-25abc135a3c8)

**Comment by luca-angemi** [comment]
they might ban us all now :laughing:

**Comment by luca-angemi** [comment]
New data are coming in since last time around 3 AM! 🎉

**Comment by sirdir1972** [comment]
The Accumulated Consumption sensor started working again for me, but the others are in state ‚unknown‘. I’m I the only one?

**Comment by ldotlopez** [comment]
Hi,

First, sorry about not answering latest issues.I have had some problems catching up on some areas due to personal overload.

Now, about this bug. Issues from i-de.es servers are a bit frequent and there is nothing we can do about it.

I'm thinking on opening a "pinned" bug with "wont-fix" label about no data comming from i-de.es and comments open, something like a little forum to discuss current status of upstream data. If something scalates to a real bug we can always open a specific bug.

I listen to your feedback and thoughts.

**Comment by ldotlopez** [comment]
> The Accumulated Consumption sensor started working again for me, but the others are in state ‚unknown‘. I’m I the only one?

Historical sensors will be always in a 'unknow' state, see the FAQ: https://github.com/ldotlopez/ha-ideenergy/blob/main/FAQ.md#q-historical-sensors-consumption-and-generation-doesnt-have-a-value

**Comment by sirdir1972** [comment]
Hi!
Sorry, never noticed that before the problems arose.

**Comment by sirdir1972** [comment]
Sorry, me again. Is it working for you guys as normal now? For me it worked one day or so and since then I haven’t seen any update. 
I didn’t see anything in the log either. I’ve switched on debug logging now, I’ll see weather that shows anything.

**Comment by hartmood** [comment]
Yes, it worked well since then. I also got a combination from Iberdrola, that they have fixed it.
